osu.Game/Database/ArchiveModelManager.cs

```diff
@@ -218,8 +220,8 @@ public void Delete(List<TModel> items)
                         // user requested abort
                         return;
 
-                    notification.Text = $"Deleting ({i} of {items.Count})";
-                    notification.Progress = (float)++i / items.Count;
+                    notification.Progress = (float)i / items.Count;
+                    notification.Text = $"Deleting ({++i} of {items.Count})";

```

Comment:
Almost the same thing as mentioned above, progress would be 0 for the first map. How about the following sequence instead:

```
notification.Text = "Deleting {++i} of ...";
Delete(b);
notification.Progress = (float)i / items.Count;
```

Reply:
is this even visible?